examples/sample-app/README.md

```diff
@@ -99,48 +129,90 @@ All commands assume the `openshift` binary is in your path (normally located und
+
+        $ osc namespace test
 
-        $ openshift cli create project -f project.json
+9. *Optional:* View the OpenShift web console in your browser by browsing to `https://<host>:8444`
 
-8. *Optional:* View the OpenShift web console in your browser by browsing to `https://<host>:8444`
-    If you click the `Hello OpenShift` project and leave the tab open, you'll see the page update as you deploy objects into the project and run builds.
+    * You will need to have the browser accept the certificate at
+      `https://<host>:8443` before the console can consult the OpenShift
+      API. Of course this would not be necessary with a legitimate
+      certificate.
+    * If you click the `Hello OpenShift` project and leave the tab open,
+      you'll see the page update as you deploy objects into the project
+      and run builds.
 
-9. Fork the [ruby sample repository](https://github.com/openshift/ruby-hello-world)
+10. *Optional:* Fork the [ruby sample repository](https://github.com/openshift/ruby-hello-world)
+    to an OpenShift-visible git account that you control, preferably
+    somewhere that can also reach your OpenShift server with a webhook.
+    A github.com account is an obvious place for this, but an in-house
+    git hosting site may work better for reaching your OpenShift server.
 
-10. *Optional:* Add the following webhook to your new github repository:
+    We will demonstrate building from a repository and then triggering
+    a new build from changing that repository. If you do not have an
+    account that will work for this purpose, that is fine; just use
+    a GitHub account and simulate the webhook (demonstrated below).
+    Without your own fork, you can still run the initial build from
+    OpenShift's public repository, just not a changed build.
+
+11. *Optional:* Add the following webhook under the settings in your new GitHub repository:
 
         $ https://<host>:8443/osapi/v1beta1/buildConfigHooks/ruby-sample-build/secret101/github?namespace=test
-  * Note: Using the webhook requires your OpenShift server be publicly accessible so github can reach it to invoke the hook.
 
-11. Edit application-template-stibuild.json
+  * Note: Using the webhook requires that your OpenShift server be
+    publicly accessible so GitHub can reach it to invoke the hook. You
+    will almost certainly need to "Disable SSL Verification" for your test
+    instance as the certificate chain generated is not publicly verified.
+
+12. Edit application-template-stibuild.json which will define the sample application
+
  * Update the BuildConfig's sourceURI (git://github.com/openshift/ruby-hello-world.git) to point to your forked repository.
+   *Note:* You can skip this step if you did not create a forked repository.
  * Replace occurences of `172.30.17.3` with the IP address of the docker-registry service as seen in step 5.
+   *Note:* You can skip this step if your registry service IP is 172.30.17.3, which should normally be the case.
```

Comment:
Actually it doesn't look like the template includes the registry IP anymore. Which is good, it should be picked up from the service by default, but I didn't notice Clayton nuked that a week ago!